inspektr/src/main/java/org/ccci/idm/user/inspektr/resolver/ReturnUserAuditResourceResolver.java

```diff
@@ -0,0 +1,24 @@
+package org.ccci.idm.user.inspektr.resolver;
+
+import static org.ccci.idm.user.inspektr.util.InspektrSerializationUtils.userToStringHelper;
+
+import org.aspectj.lang.JoinPoint;
+import org.ccci.idm.user.User;
+import org.jasig.inspektr.audit.spi.AuditResourceResolver;
+
+public class ReturnUserAuditResourceResolver implements AuditResourceResolver {
```

Comment:
yes, you configure which implementation you want to use paired with a specified `resourceResolverName` string constant